dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/config/WSBaseConfig.java

```diff
@@ -10,6 +10,7 @@
 
 package org.dpppt.switzerland.backend.sdk.config.ws.config;
 
+import io.jsonwebtoken.SignatureAlgorithm;
```

Comment:
Why was this added?

Reply:
In fact it just moved from the bottom of the import list to the top...

Perhaps the java-linting should have an additional import-cleanup and sorting stage.

Reply:
ah ups :D didn't see that